x/cron/keeper/keeper.go

```diff
@@ -67,19 +67,27 @@ func (k *Keeper) Logger(ctx sdk.Context) log.Logger {
 // ExecuteReadySchedules gets all schedules that are due for execution (with limit that is equal to Params.Limit)
 // and executes messages in each one
 // NOTE that errors in contract calls rollback all already executed messages
```

Comment:
do we even need this comment line? that's a pretty customary behaviour when dealing with smart contracts, whereas here that huge NOTE looks like it's some peculiarity. it would be better if it was REMEMBER, not NOTE, but still I think it's better to remove the line for clarity and brevity

Reply:
Removed